src/Tests/dotnet-watch.Tests/HotReload/UpdatePayloadTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
```

Comment:
Their repo is configured to add this. I'm not intentionally adding this (and not going to fight it any more).